_version.py

```diff
@@ -31,7 +34,7 @@
 # Reserved for mission use to denote patches/customizations as needed.
 # Values 1-254 are reserved for mission use to denote patches/customizations as # needed. NOTE: Reserving 0 and 0xFF for cFS open-source development use 
 # (pending resolution of nasa/cFS#440)
-_cFS_GrndSys_MISSIONREV = 255 
+_cFS_GrndSys_MISSIONREV = 0
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.